tools/ConnectCloudwatchDashboard/stack.yaml

```diff
@@ -68,7 +68,7 @@ Resources:
     Properties:
       Description: "Generates (to S3) the CloudFormation template to create Amazon Connect dashboard for instance."
       Handler: index.handler
-      Runtime: nodejs14.x
+      Runtime: nodejs16.x
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Deprecated runtime (nodejs16.x) specified. Updating disabled since 2024-06-12. Please consider updating to nodejs20.x